package.json

```diff
@@ -52,6 +52,7 @@
     "pretest": "npm run build",
     "test": "mocha --timeout 6000 -r ts-node/register \"tests/**/*.spec.ts\"",
     "test:fast": "npm run build:fast && npm run test",
+    "catch-uncommitted": "ts-node --type-check -P automation/tsconfig.json automation/run.ts catch-uncommitted",
     "ci": "npm run test && catch-uncommitted",
```

Comment:
Regarding the pre-existing `ci` script, it is still referred to in the `.travis.yml` file. I'm leaving it alone (just like `.travis.yml`) just in case we need/want to (temporarily) revert to Travis and AppVeyor. Just a couple of weeks ago, balena CI was unavailable for 10 days... Luckily we didn't have any urgent CLI PR during that period, but if we had, we might have reverted to Travis and AppVeyor.